pkg/cmd/pulumi/convert_test.go

```diff
@@ -47,7 +47,7 @@ func TestPclConvert(t *testing.T) {
 
 	result := runConvert(
 		env.Global(), []string{}, "pcl_convert_testdata",
-		[]string{}, "pcl", "pcl", tmp, true, true)
+		[]string{}, "pcl", "pcl", tmp, true, true, "")
```

Comment:
Should add a test to this file that sets the project name. We'll probably need to do that for a yaml conversion and pcl conversion because they're different enough code paths.